test/integration/deployment/util.go

```diff
@@ -375,3 +406,24 @@ func (d *deploymentTester) listUpdatedPods() ([]v1.Pod, error) {
 	}
 	return ownedPods, nil
 }
+
+func (d *deploymentTester) waitForReplicaSetHasDesiredReplicas(replicaset *v1beta1.ReplicaSet) error {
```

Comment:
naming nit: `waitForDesiredReplicas`.

Reply:
@janetkuo As this can be combined with `waitRSStable`, this will be renamed with the same name as `waitRSStable` too.